@navikt/core/react/src/util/focus-boundary/FocusBoundary.tsx

```diff
@@ -48,30 +40,43 @@ interface FocusBoundaryProps extends React.HTMLAttributes<HTMLDivElement> {
    */
   trapped?: boolean;
   /**
-   * Event handler called when auto-focusing on mount.
-   * Can be prevented.
+   * Will try to focus the given element on mount.
+   *
+   * If not provided, FocusBoundary will try to focus the first
+   * tabbable element inside the boundary.
    */
-  onMountAutoFocus?: (event: Event) => void;
+  initialFocus?:
+    | boolean
+    | React.MutableRefObject<HTMLElement | null>
+    | (() => boolean | HTMLElement | null | undefined);
   /**
-   * Event handler called when auto-focusing on unmount.
-   * Can be prevented.
+   * Tries to focus the previously focused element when unmounting.
+   *
+   * If not provided, FocusBoundary will try to focus the element
+   * that was focused before the FocusBoundary mounted.
```

Comment:
This description feels a bit unprecise. 

```suggestion
   * Will try to focus the given element on unmount.
   * 
   * If not provided, FocusBoundary will try to focus the element
   * that was focused before the FocusBoundary mounted.
   * 
   * Set to `false` to not focus anything.
```